client/modules/cm-lang-ftml/src/grammars/tex.yaml

```diff
@@ -31,12 +31,15 @@ repository:
     tag: string
 
   CommandGroup:
-    match: /(\\#?[a-zA-Z\d]+)(\{)([^]*?)(\})/
-    captures:
-      0: { is: Command }
-      1: { is: Bracket }
-      2: { type: CommandGroupString, tag: string }
-      3: { is: Bracket }
+    begin:
+      match: /(\\#?[a-zA-Z\d]+)(\{)/
```

Comment:
Does this highlight brackets not associated with a `\item`? You can group stuff using `{..}` without another context.

Reply:
Right... I knew that too. I'm not quite sure why I had the whole `CommandGroup` thing? I think it was about `\begin{align}` or whatever commands, but that requires actually discriminating between different commands to display properly. I'll remove this whole rule.